src/app/components/address-input/address-input.component.ts

```diff
@@ -196,7 +196,8 @@ export class AddressInputComponent
         if (!obj) {
             this.inputFieldFc.reset('', { emitEvent: false });
         } else {
-            this.inputFieldFc.setValue(obj, { emitEvent: false });
+            this._errors = null;
+            this.inputFieldFc.setValue(obj, { emitEvent: true });
```

Comment:
It is not on this part of the code but I noticed that when resolving invalid ens domains it seems the validation might not always work until you unfocus.
It seems that the `FormControl` is not in a `touched` or `dirty` state and thus the error message doesn't show.

After some quick looking into it:
  
https://github.com/raiden-network/webui/blob/4b654f34878e50c5a89af7cbb498ac6a286a1e06/src/app/components/address-input/address-input.component.ts#L143

Adding the following line directly below, seems to fix the problem.
```typescript
    this.inputFieldFc.markAsTouched();
```
Can you have a look? there is a valid ens domain that is `kelsos86.test` on rinkeby for an invalid input you can you any domain looking string.

Reply:
I checked and doing it in your proposed way leads to showing an error message while the user is typing an address.
I think it would be better this way:
```typescript
if (value.errors.unableToResolveEns) {
    this.inputFieldFc.markAsTouched();
}
```
What do you think?

Reply:
This is a matter of semantics. Technically while you are typing your input, it is invalid until you fully type a valid one. If you have a valid address and then delete some characters it should show an error.

I would prefer an instant validation feedback, but as long as it works after you stop typing for the first time, I am also OK with it.

Reply:
I agree that an instant validation feedback is more user friendly. But to be consistent we need to change the other input fields too. They also only show an error if you unfocus the first time.

Reply:
I created an issue for that as I think it should be separate from this PR: #103